src/statusTile.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+const { getConfigOption } = require('./helpers');
+
+let tooltip: Atom$Disposable = null;
+
+const getFormatOnSaveStatus = () => getConfigOption('formatOnSaveOptions.enabled') ? 'enabled' : 'disabled';
+
+const createStatusTile = () => {
+  const element = document.createElement('div');
+  element.classList.add('prettier-atom-status-tile');
+  element.classList.add('inline-block');
+  element.dataset.formatOnSave = getFormatOnSaveStatus();
+
+  element.appendChild(document.createTextNode('Prettier'));
+
+  return element;
+};
+
+const updateStatusTile = (disposable: Atom$Disposable, element: HTMLElement) => {
+  // eslint-disable-next-line no-param-reassign
+  element.dataset.formatOnSave = getFormatOnSaveStatus();
+
+  if (tooltip) {
+    tooltip.dispose();
```

Comment:
Does the tooltip also get disposed if prettier-atom is deactivated? I think we might want to export a function from this module called something like `deactivateTooltip` and then make sure we call that in `deactivate`? This would also allow you to mock this function with Jest when testing, so that we don't have to call `createStatusTile` in a `udpateStatusTile` test (which would prevent overlapping tests).